modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main_view/props_transformer/utils/displayUndoDeleteSuccessToast.ts

```diff
@@ -28,6 +29,13 @@ export default function displayUndoDeleteSuccessToast(
 			`<strong><a href="${Liferay.ThemeDisplay.getPathFriendlyURLPublic() + url}" class=${className}><u>`,
 			'</u></a></strong>'
 		),
+		renderData: {
+			__reactDOMFlushSync: true,
```

Comment:
Is this actually necessary? In the examples you linked, explicitly needed react to behave synchronously because of some AUI race condition.